blog/en-us/dubbo-fescar.md

```diff
@@ -149,11 +149,12 @@ dubbo-storage-service.xml
     <property name="username" value="xxx" />
     <property name="password" value="xxx" />
 ```
-### Step 2: Create UNDO_LOG table for Fescar
+### Step 2: Create undo_log table for Fescar
 
 `UNDO_LOG` table is required by Fescar AT mode.
 
 ```sql
+-- 注意此处0.3.0+ 由之前的普通索引变更为唯一索引 `ux_undo_log`
```

Comment:
This line should be translated to English too?

Reply:
@kezhenxu94 thank you